ReactTask/src/redux/formFeatures/formFeaturesOperations.ts

```diff
@@ -66,9 +80,11 @@ const registerForm =
 
       const response = await axios.get<IProfileWithData>('/api/auth/profile');
       dispatch(getUserProfileSuccess(response.data));
+      //NotifySucces('Authorization was successful');
     } catch (error) {
       dispatch(registerError((error as Error).message));
       dispatch(getUserProfileError((error as Error).message));
+      //NotifyError('Registration completed successfully');
```

Comment:
????????

Reply:
I think it is unlikely that there will be a successful message in catch block

Reply:
Removed